components/zcash/BUILD.gn

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//build/rust/rust_static_library.gni")
+
+rust_static_library("rust_lib") {
```

Comment:
see https://github.com/brave/brave-core/pull/22870/files#r1571427488 - I don't see why we'd make a component for this instead of just putting it in brave_wallet/browser/zcash

Reply:
Issue for brave_wallet/rust dir https://github.com/brave/brave-browser/issues/37731